homeassistant/components/tautulli/__init__.py

```diff
@@ -50,14 +50,18 @@ def __init__(
         self,
         coordinator: TautulliDataUpdateCoordinator,
         description: EntityDescription,
+        user: PyTautulliApiUser | None = None,
     ) -> None:
         """Initialize the Tautulli entity."""
         super().__init__(coordinator)
+        entry_id = coordinator.config_entry.entry_id
+        self._attr_unique_id = f"{entry_id}_{description.key}"
```

Comment:
No, this is handled in Sensor setup. But we can move it to here if that's better.

Reply:
Yeah, just saw that <https://github.com/home-assistant/core/pull/71712/files#diff-834bd97afd93d253b6b5a82bafa8f83937d02e39da5a5554cc9a4ff5ac309e40R276>
Add a init to the session sensor class to set the unique ID instead of copying and overwriting the key/name in the sensor setup